pkg/plugins/plugindef/plugindef_types_gen.go

```diff
@@ -158,12 +158,12 @@ type ExtensionsLink struct {
 	// Description for the rendered link
 	Description string `json:"description"`
 
+	// Target where the link will be rendered
+	Id string `json:"id"`
```

Comment:
This would usually be called `ID` in Go rather than `Id`, but I'm not sure if there's currently a way to express this in the cue file. There's also been some refactoring recently ([#62777](https://github.com/grafana/grafana/pull/62777)) about renaming other instances of `Id` to `ID` so I think we should stick to `ID` if possible.

Reply:
Yeah, I know and I agree. But it is generated from the cue files so I guess we need to add something in the code generation tooling to manage this.

Reply:
@sdboyer do you know if I can control this from the cue file in some way?

Reply:
We can fix that later if needed

Reply:
We fixed it by renaming ID -> placement